content/attacks/posts/2020-09-25-KuCoin.md

```diff
@@ -0,0 +1,56 @@
+* 1,008 BTC ($10,758,404.86)
+* 11,543 ETH ($4,030,957.90)
+* 19,834,042 USDT-ETH ($19,834,042.14)
+* 18,495,798 XRP ($4,254,547.54)
+* 26,733 LTC ($1,238,539.89)
+* 999,160 USDT ($999,160)
+* $147M worth of ERC-20 tokens
+* $87M of Stellar tokens 
+
+## Timeline:
+
+- **September 26, 2020, 2:51 AM (UTC+8):** [KuCoin received an alert from the risk management system](https://www.kucoin.com/ru/news/en-kucoin-ceo-livestream-recap-latest-updates-about-security-incident) for the first time.  A few more abnormal transactions for ETH and other ERC-20 took place soon, all from the same ETH wallet address. The remaining balance of KuCoin's hot wallets began to decline abnormally. 
+- **September 26, 2020, 3:20 AM (UTC+8):** The KuCoin operation team urgently closed the server of the wallet. There were still cases of abnormal transactions after the shutdown. 
+- **September 26, 2020, 4:20 AM (UTC+8):** The KuCoin wallet team started to transfer the remaining assets from the hot wallet to cold storage. KuCoin started to investigate the incident.
+- **September 26, 2020, 4:50 AM (UTC+8):** The KuCoin wallet team transferred most of the remaining assets from the hot wallet to cold storage.
+- **September 26, 2020, 10:41 AM (UTC+8):** The KuCoin team released [the official announcement about the security incident](https://www.kucoin.com/ru/news/en-kucoin-security-incident-update).
+- **September 26, 2020, 1:56 PM (UTC+8):** KuCoin [offered rewards of up to $100,000](https://www.kucoin.com/ru/news/en-kucoin-ceo-livestream-recap-latest-updates-about-security-incident) to anyone who could provide valid information to them regarding the incident.
+- **September 29, 2020:** [The hacker has sold $19.5 million worth of tokens via decentralized exchanges (dex) platforms](https://www.elliptic.co/blog/kucoin-thief-sells-out-millions-in-crypto-tokens-on-decentralized-exchanges) like Uniswap, Kyber Network, Tokenlon.
+- **October 3, 2020, 5:51 PM (UTC+8):** KuCoin CEO and founder Johnny Lyu [claimed](https://twitter.com/lyu_johnny/status/1312359615091277824) $204 million in cryptocurrency had been recovered as of October 3 and said perpetrators had been caught. KuCoin came back to full functionality.
+- **November 11, 2020:** [KuCoin recovered 84% ($239.45 million) of stolen assets](https://cointelegraph.com/news/kucoin-recovered-84-of-stolen-crypto-after-280m-hack-says-co-founder). The remaining part, about $45.55 million (16%), [was covered by the KuCoin insurance fund](https://www.kucoin.com/ru/news/en-a-letter-from-kucoin-ceo-2020-2021-and-beyond).
+
+## Security Failure Causes
+
+On the official website, the KuCoin team [explained](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/) that the hack was made because of a leakage of the private keys of KuCoin hot wallets. The leakage could became possible due to [several reasons](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/):
```

Comment:
> On the official website, the KuCoin team [explained](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/) that the hack was made because of a leakage of the private keys of KuCoin hot wallets. The leakage could became possible due to [several reasons](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/):

The same source link used two times in one sentence, and four in the article, leave just one, in most suitable sentence